### ISSUE TITLE
forge couple do not work

### ISSUE THREAD

**Issue by charlumau** [Haoming02/sd-forge-couple#46]
I have downloaded this extension but it seem to do nothing. it would generate the same image not matter if the extension was enable or disable.

**Comment by Haoming02** [comment]
Well... any examples?

**Comment by charlumau** [comment]
> Well... any examples?

I try to generate the page had for ecample by taking the prompt. the first one was forge couple enable the other not @
![00044-1764662897](https://github.com/user-attachments/assets/ff395709-e95b-4c44-9a29-74d459b3128f)
![00045-1764662897](https://github.com/user-attachments/assets/45107b4d-bdf9-4d9a-b7fe-3995965582fa)

**Comment by Haoming02** [comment]
Switch to **Horizontal** `Tile Direction`

**Comment by charlumau** [comment]
> Switch to **Horizontal** `Tile Direction`

it did nothing
![00062-1764662897](https://github.com/user-attachments/assets/a1ab6035-8c08-462c-9deb-55d16785d411)

**Comment by Haoming02** [comment]
bruh, you changed the prompt. Set `Global Effect` to **First Line**

**Comment by Haoming02** [comment]
Read the README pls

**Comment by charlumau** [comment]
> bruh, you changed the prompt. Set `Global Effect` to **First Line**

still nothing
![00066-1764662897](https://github.com/user-attachments/assets/46fbe020-1571-4dd4-bf49-3220a9daa6c9)

**Comment by charlumau** [comment]
> Read the README pls

I did

**Comment by Haoming02** [comment]
Oh, you're using the new Forge?

Then it's probably not compatible, since they just changed the backends.

**Comment by charlumau** [comment]
> Oh, you're using the new Forge?
> 
> Then it's probably not compatible, since they just changed the backends.

ok, will this issue be fix?

**Comment by Haoming02** [comment]
We'll have to wait. It's still in experimental phase. 

https://github.com/lllyasviel/stable-diffusion-webui-forge?tab=readme-ov-file#under-construction

**Comment by Haoming02** [comment]
Please give the [`revamp`](https://github.com/Haoming02/sd-forge-couple/tree/revamp) branch a try. 

Should work for the new Forge now.

**Comment by rktvr** [comment]
> Please give the [`revamp`](https://github.com/Haoming02/sd-forge-couple/tree/revamp) branch a try.
> 
> Should work for the new Forge now.

i'll try it out and let you know

**Comment by rktvr** [comment]
revamp works for me for sd/sdxl models. doesn't work on Flux models but tbh i kinda expected that

**Comment by Haoming02** [comment]
Welp, for `Flux`, I will wait for the original Comfy node to support it...

**Comment by charlumau** [comment]
> Please give the [`revamp`](https://github.com/Haoming02/sd-forge-couple/tree/revamp) branch a try.
> 
> Should work for the new Forge now.

the link you give brought me to a error 404 page. was the revamp branch deleted?

**Comment by Haoming02** [comment]
It has been merged into `main`

**Comment by charlumau** [comment]
> It has been merged into `main`

ok, I will go try it

**Comment by charlumau** [comment]
> It has been merged into `main`

it work. congratulation

**Comment by gleverstein** [comment]
> It has been merged into `main`

oh hey there! hope you doing great! I know its been awhile (more than a year from last message here) and the problem already solved but,
I got the same problem with forge-couple while using forge. and I have no idea why it does nothing. I already updated the extension to the latest version (4.0.4) and am not using any extension with "dynamic" in their name but...nothing happens. I feel totally stuck.
